data/strings/strings.txt

```diff
@@ -16326,6 +16326,234 @@
+    uk = 1. Відкрийте «Налаштування»
+    vi = 1. Mở Thiết lập
+    zh-Hans = 1. 打开设置
+    zh-Hant = 1. 開啟設定
+
+  [location_services_disabled_on_device_2]
+    tags = ios
+    en = 2. Select Privacy
+    af = 2. Kies Privaatheid
+    ar = 2. حدد الخصوصية
+    az = 2. Məxfilik seçin
+    be = 2. Абярыце Прыватнасць
+    bg = 2. Изберете Privacy
+    ca = 2. Seleccioneu Privadesa
+    cs = 2. Vyberte možnost Soukromí
+    da = 2. Vælg Privatliv
+    de = 2. Datenschutz auswählen
+    el = 2. Επιλέξτε Απόρρητο
+    es = 2. Selecciona Privacidad
+    es-MX = 2. Seleccione Privacidad
+    et = 2. Valige Privaatsus
+    eu = 2. Hautatu Pribatutasuna
+    fa = 2. Privacy را انتخاب کنید
+    fi = 2. Valitse Tietosuoja
+    fr = 2. Sélectionne Privacy
+    he = 2. בחר פרטיות
+    hi = 2. गोपनीयता का चयन करें
+    hu = 2. Válassza az Adatvédelem
+    id = 2. Pilih Privasi
+    it = 2. Seleziona Privacy
+    ja = 2.プライバシーを選択する
+    ko = 2. 2. 개인정보 보호 선택
+    lt = 2. Pasirinkite Privatumas
+    mr = 2. गोपनीयता निवडा
+    nb = 2. Velg Personvern
+    nl = 2. Selecteer Privacy
+    pl = 2. Wybierz Prywatność
+    pt = 2. Selecciona Privacidade
+    pt-BR = 2. Selecione Privacidade
+    ro = 2. Selectați Confidențialitate
+    ru = 2. Выберите Конфидециальность
+    sk = 2. Vyberte položku Ochrana osobných údajov
+    sv = 2. Välj Sekretess
+    sw = 2. Chagua Faragha
+    th = 2. เลือกความเป็นส่วนตัว
+    tr = 2. Gizlilik'i seçin
+    uk = 2. Виберіть "Конфіденційність
+    vi = 2. Chọn Quyền riêng tư
+    zh-Hans = 2.选择隐私
+    zh-Hant = 2. 選擇隱私
+
+  [location_services_disabled_on_device_3]
+    tags = ios
+    en = 3. Select Location Services
+    af = 3. Kies Liggingdienste
+    ar = 3. حدد خدمات الموقع
+    az = 3. Məkan Xidmətləri seçin
+    be = 3. Выберыце Службы месцазнаходжання
+    bg = 3. Изберете Услуги за местоположение
+    ca = 3. Seleccioneu Serveis d'ubicació
+    cs = 3. Vyberte možnost Služby určování polohy
+    da = 3. Vælg Placeringstjenester
+    de = 3. Wähle Standortdienste
+    el = 3. Επιλέξτε Υπηρεσίες τοποθεσίας
+    es = 3. Selecciona Servicios de Localización
+    es-MX = 3. Seleccione Servicios de ubicación
+    et = 3. Valige Asukohateenused
+    eu = 3. Hautatu Kokapen zerbitzuak
+    fa = 3. Location Services را انتخاب کنید
+    fi = 3. Valitse Sijaintipalvelut
+    fr = 3. Sélectionne les services de localisation
+    he = 3. בחר שירותי מיקום
+    hi = 3. स्थान सेवाएँ चुनें
+    hu = 3. Válassza a Helymeghatározó szolgáltatások
+    id = 3. Pilih Layanan Lokasi
+    it = 3. Seleziona Servizi di localizzazione
+    ja = 3.位置情報サービスを選択する
+    ko = 3. 3. 위치 서비스 선택
+    lt = 3. Pasirinkite Vietos nustatymo paslaugos
+    mr = 3. स्थान सेवा निवडा
+    nb = 3. Velg Stedstjenester
+    nl = 3. Selecteer Locatiediensten
+    pl = 3. Wybierz Usługi lokalizacji
+    pt = 3. Selecciona Serviços de localização
+    pt-BR = 3. Selecione Serviços de localização
+    ro = 3. Selectați Location Services (Servicii de localizare)
```

Comment:
Again, let's use the same formality in all the strings: `3. Selectează Servicii de localizare`.